youtube_dl/extractor/viki.py

```diff
@@ -145,3 +145,46 @@ def _get_subtitles(self, video_id, info_webpage):
                 'ext': 'vtt',
             }]
         return res
+
+
+class VikiShowIE(InfoExtractor):
+    IE_NAME = 'viki:show'
```

Comment:
I think this should be renamed to channel or container (for what `c` stands for).